workshops/maze/src/final/main.js

```diff
@@ -0,0 +1,19 @@
+var maze;
+var mazeImg;
+
+function setup() {
+  createCanvas(200,200);
+
+  maze = createSprite(100, 100);
+  mazeImg = loadImage("http://i.filmot.org/7xDfW51.png");
```

Comment:
But then people who have an `imgur` block at their school won't be able to play this?